pages/SignIn.jsx

```diff
@@ -1,13 +1,117 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet, View, Image } from "react-native";
+import { TextInput, Button, Text } from 'react-native-paper';
+import { useState } from "react";
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import * as Crypto from 'expo-crypto';
+
+export default function SignIn({ navigation }) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSignIn = async () => {
+    try {
+      const storedUser = await AsyncStorage.getItem(username);
```

Comment:
Personnellement je gère comme ça la récupération des users qui existent en in memory : 
``` js
const currentUsersList = await AsyncStorage.getItem('users');
const users = currentUsersList ? JSON.parse(currentUsersList) : [];
```
Car la dans ton cas tu vas chercher l'item qui correspond au nom du user, ce qui n'existe pas